### DISCUSSION TITLE
If Movie is Not In Collection Remove Label

### DISCUSSION THREAD

**Discussion by dannyvfilms** [meisnate12/Plex-Meta-Manager#482] [General]
I'm using two steps to create a collection since I wasn't sure if I could apply smart collection filters to a Trakt collection. The goal was to grab the most watched movies of the year, but only show the ones the user hasn't seen, sorted by release date. Here's that setup:

Hidden Collection:
    trakt_watched_yearly: 100
    sync_mode: sync
    radarr_add: true
    item_label: watched_yearly
    collection_mode: hide
    collection_order: custom
    visible_library: false
    visible_home: false
    visible_shared: false

Public Collection:
    label: watched_yearly
    unplayed: true
    release: 365

What I just realized is that movies are going to age out of the collection. I'm not sure how to ensure that only the movies in this collection have `label: watched_yearly` and no other movies. Is there a process for that?

**Comment by meisnate12** [comment]
use a `plex_search`

```yaml
Public Collection:
  plex_search:
    all:
      label: watched_yearly
      unplayed: true
```

But this will bring about another problem that plex_search always uses the server admins stuff like unplayed that doesn't charge per user

Also smart_label collections are what your after to make a trakt_list directly into a smart collection. (Basically, it just adds a label to all of them and creates a smart collection looking for that label)

**Reply by dannyvfilms** [reply]
Well that's absolutely frustrating. I just verified on my fiancé's account, `unplayed` looks at my account. So much for "smart" collections...

**Reply by meisnate12** [reply]
I mean they're still a lot smarter then what we had before lol so I'm not too mad at it
